### ISSUE TITLE
Error copying from code section in CHM

### ISSUE THREAD

**Issue by DougHennig** [RickStrahl/HelpBuilderReleases#56]
Not sure there's anything you can do about this, but if I try to copy a code sample that's surrounded with ``` in the CHM, either with Ctrl+C or right-click and Copy, I get the following error:

![aceerror](https://cloud.githubusercontent.com/assets/12052409/18558090/5a65c012-7b37-11e6-9c06-faec71ed25cc.png)

**Comment by RickStrahl** [comment]
Have you updated your templates recently? There were issues with this a while back related to edge mode usage in the templates. For release I will likely switch to highlightjs for highlighting which should remove some of this weirdness as well as making the help files load faster.

**Comment by RickStrahl** [comment]
Doug I was able to verify that this is broken. I switched over to highlightjs for rendering in the latest builds and the problem is eleviated with that. 

WHen the next build arrives make sure you update the templates. There will be some changes in what files are used and referenced so _layout.wcs will need to be updated (or updated from templates, then any changes merged back in from _layout_old.wcs.
